.github/workflows/ci.yml

```diff
@@ -99,7 +104,10 @@ jobs:
   release:
     name: Release
     needs: build
-    if: startsWith(github.ref, 'refs/tags/v') || (github.event_name == 'workflow_dispatch' && github.event.inputs.publishSnapshot == 'true') # || (github.ref == 'refs/heads/main')
+    if:
+      startsWith(github.ref, 'refs/tags/v') ||
+      github.event.inputs.publishSnapshot ||
```

Comment:
note: I removed the `event_name == workflow_dispatch` check, as I think it's redundant - at the moment, inputs can only be set by running a manual dispatch. In other jobs, the input defaults to false.